scala-package/core/src/main/scala/ml/dmlc/mxnet/Initializer.scala

```diff
@@ -33,6 +33,10 @@ abstract class Initializer {
 
     if (name.startsWith("upsampling")) {
       initBilinear(name, arr)
+    } else if (name.startsWith("stn_loc") && name.endsWith("weight")) {
```

Comment:
we don't do this anymore.
name matching is a bad idea. use Variable(..., init=xx) instead